modules/space/rendering/renderablefluxnodes.cpp

```diff
@@ -0,0 +1,868 @@
+ ****************************************************************************************/
+
+#include <modules/space/rendering/renderablefluxnodes.h>
+
+#include <openspace/engine/globals.h>
+#include <openspace/engine/windowdelegate.h>
+#include <openspace/navigation/navigationhandler.h>
+#include <openspace/navigation/orbitalnavigator.h>
+#include <openspace/rendering/renderengine.h>
+#include <openspace/scene/scene.h>
+#include <openspace/util/timemanager.h>
+#include <openspace/util/updatestructures.h>
+#include <openspace/query/query.h>
+#include <ghoul/filesystem/filesystem.h>
+#include <ghoul/filesystem/cachemanager.h>
+#include <ghoul/logging/logmanager.h>
+// Test debugging tools more then logmanager
+#include <ghoul/logging/consolelog.h>
+#include <ghoul/logging/visualstudiooutputlog.h>
+#include <ghoul/opengl/programobject.h>
+#include <ghoul/opengl/textureunit.h>
+#include <fstream>
+#include <functional> 
+#include <optional>
+#include <sys/stat.h>
+#include <thread>
+
+namespace {
+    // log category
+    constexpr const char* _loggerCat = "RenderableFluxNodes";
+    
+    //streamColor, nodeSize, nodeSizeLargerFlux, thresholdFlux, 
+    constexpr const std::array<const char*, 29> UniformNames = {
+        "streamColor", "nodeSize", "proximityNodesSize", 
+        "thresholdFlux", "colorMode", "filterLower", "filterUpper", "scalingMode",
+        "colorTableRange", "domainLimZ", "nodeSkip", "nodeSkipDefault", "nodeSkipEarth",
+        "nodeSkipMethod", "nodeSkipFluxThreshold", "nodeSkipRadiusThreshold",
+        "fluxColorAlpha", "earthPos", "distanceThreshold", "time", "maxNodeDistanceSize", 
+        "usingCameraPerspective", "drawCircles", "drawHollow", "useGaussian",
+        "perspectiveDistanceFactor", "minMaxNodeSize", "usingPulse",
+        "usingGaussianPulse"
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo GoesEnergyBinsInfo = {
+        "GoesEnergy",
+        "GOES Energy",
+        "Select which energy bin you want to show. GOES = Geostationary Operational "
+        "Environmental Satellites Emin01 is values > 10 Mev, "
+        "Default is Emin03 where values > 100 Mev."
+    };
+    constexpr openspace::properties::Property::PropertyInfo ColorModeInfo = {
+        "ColorMode",
+        "Color Mode",
+        "Color lines uniformly or using color tables based on specific values on nodes, "
+        "for examples flux values."
+    };
+    constexpr openspace::properties::Property::PropertyInfo ColorTablePathInfo = {
+        "ColorTablePath",
+        "Path to Color Table",
+        "Color Table/Transfer Function to use for 'By Flux Value' coloring."
+    };
+    constexpr openspace::properties::Property::PropertyInfo StreamColorInfo = {
+        "Color",
+        "Color",
+        "Color of particles."
+    };
+    constexpr openspace::properties::Property::PropertyInfo NodeSizeInfo = {
+       "NodeSize",
+       "Size of nodes",
+       "Change the size of the nodes"
```

Comment:
"The size of the nodes"

But it is actually the size of the rendered points for the nodes it seems like? So that should be clarified :)